adhocracy4/comments/static/comments/Comment.jsx

```diff
@@ -8,12 +8,15 @@ var CommentManageDropdown = require('./CommentManageDropdown')
 
 var React = require('react')
 var django = require('django')
-var moment = require('moment')
 
 var safeHtml = function (text) {
   return { __html: text }
 }
 
+var localeDate = function (dateStr) {
+  return new Date(dateStr).toLocaleString(document.documentElement.lang)
```

Comment:
Actually our format does not use a space. Outputting `this.props.created` gives me `2017-08-24T13:20:14.642234Z`.